### DISCUSSION TITLE
Can I use GitHub private repo to host a GitHub pages to a tiny company?

### DISCUSSION THREAD

**Discussion by rafaeor** [community/community#74978] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey, I would like to know if it's according to the rules a website hosting(simple html page) using GitHub pages.

I tried to search for a rule, however I did not find anything about it. 
What I saw was "2K or 3K GitHub Actions minutes per month", which I'm not sure what that mean.

**Comment by ktauchathuranga** [comment]
"It is only possible with GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server."

[GitHub Pages for Private Repository [closed]](https://stackoverflow.com/questions/28744980/github-pages-for-private-repository)

**Comment by ediawa** [comment, marked as answer]
it is great option for small companies that want to have control over code and limit access on their website's source files
